File: application/src/main/java/hu/bsstudio/Application.java
package hu.bsstudio;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "hu.bsstudio")
public class Application {

    public static void main(final String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: app/src/main/kotlin/hu/bsstudio/robonaut/Application.kt
package hu.bsstudio.robonaut

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = [ "hu.bsstudio.robonaut" ])
class DemoApplication

fun main(args: Array<String>) {
    runApplication<DemoApplication>(*args)
}
